velox/vector/arrow/Bridge.cpp

```diff
@@ -726,7 +726,8 @@ void exportToArrow(const VectorPtr& vec, ArrowSchema& arrowSchema) {
 }
 
 TypePtr importFromArrow(const ArrowSchema& arrowSchema) {
-  const char* format = arrowSchema.format;
+  const char* format = arrowSchema.dictionary ? arrowSchema.dictionary->format
```

Comment:
can we have unit tests for this change. This should go under `ArrowBridgeSchemaTest.cpp`